science/bali-phy/Portfile

```diff
@@ -1,110 +1,39 @@
 PortSystem 1.0
-
+#PortGroup           github 1.0
 name                bali-phy
-version             2.1.1
-revision            5
-description         Bayesian Alignment and Phylogeny estimation
-long_description    BAli-Phy can estimate phylogenetic trees from sequence data when the alignment \
-                    is uncertain. Instead of conditioning on a single alignment estimate, BAli-Phy \
-                    accounts for alignment uncertainty by integrating over all alignments. \
-                    BAli-Phy does not rely on a guide tree because the alignment and the tree are \
-                    co-estimated. Therefore it can construct phylogeny estimates of widely \
-                    divergent sequences without bias toward a guide tree.
+version             3.0-beta6-20171223
+revision            1
```

Comment:
Revision should remain zero until we commit the changes.

Reply:
OK.